test/MUXControlsTestApp/master/DatePicker.xml

```diff
@@ -0,0 +1,173 @@
+[Windows.UI.Xaml.Controls.Grid]
+    Padding=0,0,0,0
+    CornerRadius=0,0,0,0
+    BorderThickness=0,0,0,0
+    BorderBrush=[NULL]
+    Background=[NULL]
+    Width=400
+    Margin=0,0,0,0
+    Height=400
+    FocusVisualSecondaryThickness=1,1,1,1
+    FocusVisualSecondaryBrush=#99FFFFFF
+    FocusVisualPrimaryThickness=2,2,2,2
+    FocusVisualPrimaryBrush=#FF000000
+    Visibility=Visible
+    RenderSize=400,400
+  [Windows.UI.Xaml.Controls.DatePicker]
+      Padding=0,0,0,0
+      Foreground=#FF000000
+      BorderThickness=0,0,0,0
+      BorderBrush=[NULL]
+      Background=[NULL]
+      MinWidth=296
```

Comment:
Why do we only have Min/MaxWidth here and not in DatePicker-7?

Reply:
I'm not sure...

We never set Min/MaxWidth on DatePicker in xaml. The values are set on its children HeaderContentPresenter and FlyoutButton, not sure how DatePicker itself got those values. I'll double check...